### PR TITLE
Added a codecov.yml configuration file

### LINKED ISSUE #159: Add a codecov.yml file to better configure how codecov works
`codecov.yml` is a YAML File to configure how [codecov](https://codecov.io/) works. You're supposed to put it in the root directory of your repository. Documentation:

https://github.com/codecov/support/wiki/codecov.yml

### PR DESCRIPTION
**Pull request by ttmc**
This PR adds a first cut at a `codecov.yml` file. I gather that the `codecov.yml` file is fairly new, and the documentation about it isn't great. The documentation that does exist is at:

https://github.com/codecov/support/wiki/codecov.yml

The file was created by starting with the default and then adding a few things:

* branch: develop
* bot: r-marques
* ignore: (list of directories to ignore)
* Added "sunburst" to comment: layout: because a sunburst sounds neat

Resolves #159

### CODE REVIEW

File: codecov.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+codecov:
+  branch: develop    # the branch to show by default
+
+  # The help text for bot says:
+  # "the username that will consume any oauth requests
+  # must have previously logged into Codecov"
+  # In GitHub - BigchainDB organization settings - Third-party access,
+  # it says, for Codecov: "approval requested by r-marques"
+  bot: r-marques
```
Comment (r-marques) [inline codecov.yml]: why that guy?
should we use @wrigley instead?

**Comment by r-marques** [comment]
I say we could try this and change it after if needed. The coverage is not working correctly as is.
Make this branch up to date with develop and then merge :+1:

File: codecov.yml (hunk 1 above)
Comment (ttmc) [inline codecov.yml]: I don't know if @wrigley has previously logged into Codecov, and I have no way of checking.

**Comment by ttmc** [comment]
Okay I made the branch up to date with develop and will now merge...

**Pull request merged**